lib/evmone/eof.cpp

```diff
@@ -349,9 +376,30 @@ std::variant<EOFValidationError, int32_t> validate_max_stack_height(
             stack_height_change =
```

Comment:
Consider the called function can be a Non Returning Function (0x80). In this case the number `0x80` should not be considered as the total number of outputs.

Reply:
I fixed this now by treating CALLF to non-returning as if it has 0 outputs.

But I'm thinking it would be better to disallow CALLF to non-returning altogether (so that only JUMPF is allowed)